stl/inc/vector

```diff
@@ -2882,6 +2882,64 @@ namespace pmr {
     using vector = _STD vector<_Ty, polymorphic_allocator<_Ty>>;
 } // namespace pmr
 #endif // _HAS_CXX17
+
+#if _HAS_IF_CONSTEXPR
+// VARIABLE TEMPLATE _Is_vb_iterator
+template <class _Alloc>
+_INLINE_VAR constexpr bool _Is_vb_iterator<_Vb_const_iterator<_Alloc>> = true;
+
+template <class _Alloc>
+_INLINE_VAR constexpr bool _Is_vb_iterator<_Vb_iterator<_Alloc>> = true;
+
+template <class _FwdIt, class _Ty>
+_CONSTEXPR20 void _Fill_vbool(_FwdIt _First, _FwdIt _Last, const _Ty& _Val) {

```

Comment:
Why not 2 vs 1 wins

Reply:
Urg, can I take that back. I just rebased the implementations for `count` and `find`

While for `count` We could simply use `_Vb_const_iterator` for `find` we need to return the passed iterator type. So can we go back to `_FwdIt` as that is always the correct thing to pass?

Reply:
Ah, I see. Yeah, that makes sense. (In vNext, I've overhauled the iterators to make constness a `bool` parameter, instead of the current scheme of having different types that inherit.)